pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/pubsub/UpdateWorkflowStatusTest.java

```diff
@@ -21,77 +23,84 @@
+    public static final String UPDATE_WORKFLOW_TEST = "UpdateWorkflowTest";
     public static final String RGP = "RGP";
-    private static final String participantId = "RBMJW6ZIXVXBMXUX6M3Q";
-    private static final Map<String, String> testParticipantData = new HashMap<>();
+    private static final Map<String, String> participantDataMap = new HashMap<>();
     private static final DDPInstanceDao ddpInstanceDao = new DDPInstanceDao();
     private static final FieldSettingsDao fieldSettingsDao = FieldSettingsDao.of();
     private static final UserDao userDao = new UserDao();
     private static final ParticipantDataDao participantDataDao = new ParticipantDataDao();
     private static Gson gson;
     private static UserDto userDto;
+    private static String participantId;
     private static int participantDataId;
     private static ParticipantData participantData;
     private static DDPInstanceDto ddpInstanceDto;
+    private static String instanceName;
+    private static int fieldSettingsId;
 
     @BeforeClass
-    public static void doFirst() {
-        setupDB();
+    public static void setup() {
         gson = new Gson();
 
-        ddpInstanceDto = DBTestUtil.createTestDdpInstance(ddpInstanceDto, ddpInstanceDao, "UpdateWorkflowInstance");
-
+        instanceName = String.format("%s_%d", UPDATE_WORKFLOW_TEST, Instant.now().toEpochMilli());
+        ddpInstanceDto = DBTestUtil.createTestDdpInstance(ddpInstanceDao, instanceName);
         userDto = DBTestUtil.createTestDsmUser("UpdateWorkflowUser", "UpdateWorkflow@status.com", userDao, userDto);
 
-        createDataForParticipant();
-
         createParticipantData();
-
-    }
-
-    private static void createDataForParticipant() {
-        testParticipantData.put("REGISTRATION_STATUS", "REGISTERED");
-        testParticipantData.put("MEMBER_TYPE", "SELF");
-    }
-
-    private static void createParticipantData() {
-        participantData =
-                new ParticipantData.Builder().withDdpParticipantId(participantId).withDdpInstanceId(ddpInstanceDto.getDdpInstanceId())
-                        .withFieldTypeId(UPDATE_WORKFLOW_TEST).withData(gson.toJson(testParticipantData))
-                        .withLastChanged(System.currentTimeMillis()).withChangedBy(userDto.getEmail().orElse("")).build();
-        participantDataId = participantDataDao.create(participantData);
-        participantData = new ParticipantData.Builder().withParticipantDataId(participantDataId).withDdpParticipantId(participantId)
-                .withDdpInstanceId(ddpInstanceDto.getDdpInstanceId()).withFieldTypeId(UPDATE_WORKFLOW_TEST)
-                .withData(gson.toJson(testParticipantData)).withLastChanged(System.currentTimeMillis())
-                .withChangedBy(userDto.getEmail().orElse("")).build();
+        createFieldSettings();
     }
 
     @AfterClass
-    public static void finish() {
+    public static void tearDown() {
```

Comment:
The other pattern I've seen us use in our code is to rollback.  GBFOrderFinderTest.java has an example.  The code here implies that we accept state crossing between two tests.  Is that intentional?

Reply:
I think this will work because we will always have unique row IDs per test instance. But I could have missed something?
I'm not sure how the rollback would work here since we are opening transactions in the code under test.

Reply:
I think we're good between runs across the class, but it looks like we're reusing the same participant and mutating it between tests.  This doesn't appear to be a result of anything you've changed but may lead to some flaky tests in the future.  Fine for now.

Reply:
I might have missed something, but I believe we are creating a new participant for each run of this test, and other tests are not using that user.
Is that consistent with what you see?